Library/Formula/sfk.rb

```diff
@@ -0,0 +1,22 @@
+class Sfk < Formula
+  homepage "http://stahlworks.com/dev/swiss-file-knife.html"
+  url "http://internode.dl.sourceforge.net/project/swissfileknife/1-swissfileknife/1.7.4/sfk-1.7.4.tar.gz"
```

Comment:
This should be a general Sourceforge URL rather than a specific mirror.

Example => ` https://downloads.sourceforge.net/project/ccd2iso/ccd2iso/ccd2iso-0.3/ccd2iso-0.3.tar.gz `.